examples/yolov3.py

```diff
@@ -404,4 +406,6 @@ def forward(self, x):
   show_labels(prediction)
   prediction = process_results(prediction)
   boxes = add_boxes(np.array(Image.fromarray(img).resize((608, 608))), prediction)
-  cv2.imwrite('boxes.jpg', boxes)
+  os.chdir(pathlib.Path(__file__).parent)
```

Comment:
Why?! Just the dir I ran it from, you don't need this line.

Reply:
Sorry, it was confusion what you wanted (for some reason interpreted as script directory, opps).